app/controllers/seeds_controller.rb

```diff
@@ -49,7 +49,7 @@ def new
     @seed = Seed.new
 
     # using find_by_id here because it returns nil, unlike find
-    @crop = Crop.find_by_id(params[:crop_id]) || Crop.new
+    @crop = Crop.find_by(id: params[:crop_id]) || Crop.new
```

Comment:
Its OK as is (maybe adjust comment), it was more why we can't use load_resource or similar there